DigitalLearningSolutions.Data/Services/SessionService.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Data;
+    using Dapper;
+
+    public interface ISessionService
+    {
+        int StartOrRestartSession(int candidateId, int customisationId);
+        void StopSession(int candidateId);
+        void UpdateSessionDuration(int sessionId);
+    }
+
+    public class SessionService : ISessionService
+    {
+        private readonly IDbConnection connection;
+
+        public SessionService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        private const string stopSessionsSql =

```

Comment:
Sorry missed this earlier, this should be declared above the constructor 🙂

Reply:
Fixed in 91b2354 🙂